int32 lane_change
float64 distance
geometry_msgs/Vector3 location
geometry_msgs/Vector3 rotation